attestation-agent/attestation-agent/src/bin/grpc-aa/main.rs

```diff
@@ -70,6 +83,12 @@ pub async fn main() -> Result<()> {
             }
         }
     }
+    if let Some(initdata_toml_path) = cli.initdata_toml {
```

Comment:
would it make sense to consolidate the handling of both options (toml file and digest), so make it more obvious. something like this:

```rust
let mut initdata_digest = cli.initdata_digest; 
if let Some(toml) = cli.initdata_toml {
  if initdata_digest.is_some() {
    bail!("initdata digest already provided");
  }
  let digest = calculate_initdata_digest(toml);
  initdata_digest = Some(digest);
}

if let Some(digest) = initdata_digest {
   aa.bind_init_data(digest).await?;
   ...
}
```

Reply:
Yes, you are right that we should have exclusive logic. Let me do this.

Reply:
I added some logic for the two configurations to have override priority to avoid direct errors. Relative information will be print via CLI help. PTAL